bugtool/cmd/root.go

```diff
@@ -77,6 +79,7 @@ var (
 func init() {
 	BugtoolRootCmd.Flags().BoolVar(&archive, "archive", true, "Create archive when false skips deletion of the output directory")
 	BugtoolRootCmd.Flags().BoolVar(&getPProf, "get-pprof", false, "When set, only gets the pprof traces from the cilium-agent binary")
+	BugtoolRootCmd.Flags().BoolVar(&envoyDump, "envoy-dump", false, "When set, dump envoy configuration from unix socket")
```

Comment:
can I enable this by default ? This will not stop bugtool even failure (i.e. proxy is not running)

Reply:
From my side I think we should enable by default.

Reply:
thanks, I have updated the default value :100: